crates/uv/src/cli.rs

```diff
@@ -1308,6 +1308,9 @@ pub(crate) struct PipInstallArgs {
     /// print the resulting plan.
     #[arg(long)]
     pub(crate) dry_run: bool,
+
+    #[arg(long, hide = true)]
+    pub(crate) unstable_uv_lock_file: Option<String>,
```

Comment:
Nit: can we call this `unstable_lock_file` (omitting `uv`)?

Reply:
It matches the name I added in: https://github.com/astral-sh/uv/pull/3314/files#diff-ae55eef503096c43098e973865bd5ed612871dcd1d3d335631a8027deb3828d3R605

This flag is hidden, undocumented and intended to be temporary. Its only purpose is to expose "install from `uv.lock`" on the CLI so that we can interact with it.

Reply:
I mostly just have a strong dislike for using `uv` in things (outside of crate names, since it's roughly a namespace), like structs or arguments. But not a big deal if it's temporary.